composer.json

```diff
@@ -82,6 +83,8 @@
     },
     "scripts": {
         "analyse": "vendor/bin/phpstan analyse",
+        "cs": "vendor/bin/php-cs-fixer fix -v --dry-run --rules=@Symfony src/",
+        "cs:fix": "vendor/bin/php-cs-fixer fix -v --rules=@Symfony src/",
```

Comment:
I think it's better to create a `.php-cs-fixer.dist.php` with `@Symfony` ruleset definition, so running locally no need to provide rules

Reply:
Done in 785e21c6219b9388cb4a444abf33263c78125806